local/kaltura/version.php

```diff
@@ -25,8 +25,45 @@
     die('Direct access to this script is forbidden.');
 }
 
-$plugin->version = 20210620311;
+$plugin->version = 2021051701;
 $plugin->component = 'local_kaltura';
-$plugin->release = 'Kaltura release 4.3.0';
+$plugin->release = 'Kaltura release 4.3.1';
 $plugin->requires = 2018120300;
 $plugin->maturity = MATURITY_STABLE;
+
+global $DB;
+
+$localKalturaPluginVersionRecords = $DB->get_records_select('config_plugins', "plugin = 'local_kaltura' AND name = 'version'");
+
+$kalturaPluginVersion = "";
+if ($localKalturaPluginVersionRecords) {
+    foreach ($localKalturaPluginVersionRecords as $key => $localKalturaPluginVersionRecord) {
+        $kalturaPluginVersion = $localKalturaPluginVersionRecord->value;
+        break;
+    }
+}
+
+if ($kalturaPluginVersion == 20210620311) {
+    $pluginsRecords = $DB->get_records_select('config_plugins', "name = 'version' AND value = '$kalturaPluginVersion'");
```

Comment:
I tried to put this code in a function but I got a white screen in the browser, when removed the function it was fixed